config.coffee

```diff
@@ -17,6 +17,7 @@ if !fs.existsSync('config.json')
 			secret: process.env.S3_SECRET
 			bucket: process.env.S3_BUCKET
 			directory: process.env.S3_DIRECTORY
+			url: 'http://parallellocalypse.s3-website-us-east-1.amazonaws.com/'
```

Comment:
I see. I think you need to discuss with Pablo and make it more robust with
fewer hard-coded things. Also check the code that uploads images (or is it
done by hand?)

On Mon, May 25, 2015 at 2:37 PM cPu1 <notifications@github.com> wrote:

> In config.coffee
> <https://github.com/supercomputer-io/parallellocalypse-hub/pull/6#discussion_r30975624>
> :
>
> > @@ -17,6 +17,7 @@ if !fs.existsSync('config.json')
> >  			secret: process.env.S3_SECRET
> >  			bucket: process.env.S3_BUCKET
> >  			directory: process.env.S3_DIRECTORY
> > +			url: 'http://parallellocalypse.s3-website-us-east-1.amazonaws.com/'
>
> I found this URL hard-coded on the client config here:
> https://github.com/supercomputer-io/parallellocalypse-hub/blob/master/public/js/app.coffee#L45
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/supercomputer-io/parallellocalypse-hub/pull/6/files#r30975624>
> .
>